sdk/src/main/java/com/silanis/esl/sdk/builder/SignerBuilder.java

```diff
@@ -377,6 +379,11 @@ public SignerBuilder withRoleId(String roleId) {
         return withCustomId(roleId);
     }
 
+    public SignerBuilder withLocalLanguage() {
+        this.localLanguage = "local";
+        return this;
+    }
+
```

Comment:
Do we need this local language field to use the local language? We set the local language as data in Role.
Why do we need duplicate settings?